plugins/database/mysql/mysql.go

```diff
@@ -28,11 +28,23 @@ const (
 	mySQLTypeName = "mysql"
 )
 
+// Modern
```

Comment:
Leaving this part in for future people changing these constants to know to leave some spaces for all parts.